tests/staticfiles_tests/test_management.py

```diff
@@ -188,6 +188,27 @@ def _input(msg):
             return 'yes'
         return _input
 
+    def test_no_warning_when_staticdir_does_not_exist(self):
+        stdout = six.StringIO()
+        shutil.rmtree(six.text_type(settings.STATIC_ROOT))
+        call_command('collectstatic', interactive=True, stdout=stdout)
+
+        output = force_text(stdout.getvalue())
+        self.assertNotIn(self.overwrite_warning_msg, output)
+        self.assertNotIn(self.delete_warning_msg, output)
+
+    def test_no_warning_for_empty_staticdir(self):
+        stdout = six.StringIO()
+        # Use an empty directory instead of the directory from setUp that contains files
+        static_dir = tempfile.mkdtemp(prefix='collectstatic_empty_staticdir_test')
+        with override_settings(STATIC_ROOT=static_dir):
+            call_command('collectstatic', interactive=True, stdout=stdout)
+
+        shutil.rmtree(six.text_type(static_dir))
+        output = force_text(stdout.getvalue())
+        self.assertNotIn(self.overwrite_warning_msg, output)
+        self.assertNotIn(self.delete_warning_msg, output)
+
```

Comment:
We could add a `files_copied_msg = "static files copied"`` attribute and `self.assertIn(self.files_copied_msg, output)` so that something in the output is verified.